src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -734,9 +750,12 @@ def _revYangConvert(val):
 
         # if it is a leaf-list do it for each element
         if leafDict[key]['__isleafList']:
-            vValue = list()
-            for v in value:
-                vValue.append(_revYangConvert(v))
+            if tuple(self.elementPath) in self.leaf_list_with_string_value_set:
```

Comment:
We use this lib for reverse translation as well, i.e. from yang to configdb, in that case self.leaf_list_with_string_value_set will be empty right. [i.e. only reverse translation cases.]

Reply:
What could be the case of that [only reverse translation cases]?

Reply:
Praveen provided a case that "config reload from YANG". So this is a valid concern. I will fix.